src/Equinox.Core/StopwatchInterval.fs

```diff
@@ -34,7 +34,7 @@ and [<Struct; NoEquality; NoComparison>] StopwatchInterval(startTicks : int64, e
 
 module Stopwatch =
     [<DebuggerStepThrough>]
-    let time (ct : CancellationToken) (f : CancellationToken -> Task<'T>) : Task<struct (StopwatchInterval * 'T)> = task {
+    let time (ct : 'a) (f : 'a -> Task<'T>) : Task<struct (StopwatchInterval * 'T)> = task {
```

Comment:
Suggestion: `ct` is generally taken to mean "CancellationToken" in the codebase, now that this can be anything let's change the name of the variable as well?